streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -49,15 +49,14 @@
     /** Maximum time difference for tuples that are after the join tuple. */
     public final long after;
 
-    private JoinWindows(String name, long before, long after) {
-        super(name);
-
+    private JoinWindows(long before, long after) {
+        super();
```

Comment:
No need to call `super()` explicitly -- just a question of code style. Is there any policy -- I like uniform look-and-feel :)